Added article on making PyCon India women friendly. (#217)
pythonindia/inpycon-blog: content/blog/2019-making-pycon-india-women-friendly.md

Title: Making PyCon India Women Friendly
Date: 2019-06-03
Category: 2019
Slug: 2019-making-pycon-india-women-friendly

The vision of PyCon India 2019 is to create an inclusive environment
that promotes knowledge sharing and diversity in thoughts and
ideas. To quote

<blockquote>
We would like to ensure that the conference approaches everyone with
respect, and that we resolve differences peacefully. In particular, we
will proactively extend our invite to women, provide support for
students, and ensure that people with disabilities feel welcome at the
conference.
</blockquote>

Along those lines, we would like to offer the following facilities for
women to welcome their participation:

<!-- PELICAN_END_SUMMARY -->

  * For parents travelling with young kids, childcare services will be

>     provided at the venue for kids 5 years or below. If you want to

Reviewer: ```suggestion
    provided at the venue for kids aged 5 years or below. If you want to
```

    avail childcare for your kid, please mark the checkbox for the
    same in the ticket registration page. 

  * The organizers will arrange a Kids Corner with games,

>     entertainment and activities for kids above 5 upto 12. If you want

Reviewer: ```suggestion
    entertainment and activities for kids of age 5 to 12. If you want
```
